### ISSUE TITLE
Migrate to `hfg-gmuend` controlled font-builder docker image

### ISSUE THREAD

**Issue by JeppeKlitgaard** [hfg-gmuend/openmoji#430]
At some point after #422 is merged it would be ideal to change over to use a docker image distributed through `ghcr.io` using this organisation rather than through @JeppeKlitgaard for security and maintainability purposes. There is no rush on this however.

**Comment by b-g** [comment]
Yes, let's do this once PR #422 is merged. Might need a few instructions how to set things up.

**Comment by b-g** [comment]
Running the font-builder docker image on my M1 machine yields:

```
Status: Downloaded newer image for ghcr.io/jeppeklitgaard/font_builder:latest
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

Not super familiar with the docker build pipeline ... but it would be nice to speed up the image on M1 computers. Building locally with `docker build -t <image-name> --platform linux/x86_64`apparently does the trick.

Would be it be possible to publish the image via the github action in addition as well for `--platform linux/x86_64`?

**Comment by b-g** [comment]
Seems to be possible: Multi-platform images https://docs.docker.com/build/building/multi-platform/

**Comment by JeppeKlitgaard** [comment]
There is already a GitHub workflow in #422 to build the images. Adding multi-platform support to this should be possible.

I believe we simply need to set something up in the `hfg-gmuend` organisation to support this.

Surely it wouldn't be `x86_64` for M1 Mac's but something ARM-related? Could be wrong on this.

**Comment by b-g** [comment]
Great. Should I create a repo e.g. `hfg-gmuend/openmoji-font-builder` or `hfg-gmuend/openmoji-font-buildsystem`

**Comment by JeppeKlitgaard** [comment]
I think it would be easiest if the dockerfile is hosted in this repo (currently in `helpers/docker/`), but a separate repo is definitely also an option

**Comment by b-g** [comment]
Ok, either way is fine. Let's decide once the new font buildsystem was merged.

**Comment by b-g** [comment]
PR #422 is finally merged! This is mega! Let us tackle the remaining related open [todos](https://github.com/hfg-gmuend/openmoji/issues/430#issue-1523191588) ... what should I do? :)

**Comment by JeppeKlitgaard** [comment]
Apologies for missing the ping!

I will try to take a look at this soon. I just started a new job and am still living out of a suitcase with family, so things are a bit hectic on my end at the moment. As I recall we need to do something to enable a container registry on the hfg-gmuend org, which I don't think I am able to do by myself. I will poke around a bit and we can coordinate

**Comment by b-g** [comment]
@JeppeKlitgaard No worries, not super urgent ... just a gentle ping for news :) + All the best for settling in the new place and job!

> As I recall we need to do something to enable a container registry on the hfg-gmuend org, which I don't think I am able to do by myself. I will poke around a bit and we can coordinate

Of course. Happy to setup a new repo, give you the necessary permissions ... just let me know.

**Comment by JeppeKlitgaard** [comment]
Just a heads up that this is still on my todo list, but the past few weeks have been quite hectic. Hopefully things will calm down soon!

**Comment by b-g** [comment]
Many thanks for for letting us know, that is very nice of you. We are aiming to publish OpenMoji v15 in the next weeks. All emojis have been designed to complete with Unicode 15. I just have to find time to add them to library and polish a few small other things ... But as the font-builder docker image is not a blocker, no crazy rush on our end :) All the best

**Comment by b-g** [comment]
Hi @JeppeKlitgaard, ping :) Any news? Tried it today ... bug gave up after 1h. Just not familiar enough with the nitty gritty docker details. I think I could build it via `cd helpers/dockers` and `docker buildx build -f font_builder.Dockerfile --platform linux/amd64 --tag openmoji:latest --load .` but then of course `helpers/generate-fonts.sh`was breaking ... any help would be great :) + Merry 🎄
